alerts/openshift-container-storage-operator/OSDCPULoadHigh.md

```diff
@@ -0,0 +1,46 @@
+# OSDCpuLoadHigh
+
+## Meaning
+
+This alert indicates that the CPU usage in the OSD (Object Storage Daemon)
+container on a specific pod has exceeded 80%, potentially affecting the
+performance of the OSD.
+
+## Impact
+
+OSD is a critical component in Ceph storage, responsible for managing data
+ placement and recovery. High CPU usage in the OSD container suggests increased
+ processing demands, potentially leading to degraded storage performance.
+ Creating more OSDs or adding nodes to the cluster can help distribute the load
```

Comment:
I would move the paragraph starting with "creating more OSDs.. to the "mitigation" section":
and I would rephrase it as:
Increase the number of OSDs in the cluster adding more new storage devices in the existing nodes or adding new nodes with new storage devices. This would help to distribute ....